src/components/BlankCard/index.tsx

```diff
@@ -7,7 +7,7 @@ import ImportThumbnail from "../../assets/import-thumbnail.svg";
 
 const styles = require("../ModelCard/style.css");
 
-const BlankCard: React.FunctionComponent<{}> = () => {
+const BlankCard = () => {
```

Comment:
removing the React.FunctionComponent typing?

Reply:
I found this https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/ which recommends against it too.  I do like the idea of typing the return value but it shouldn't hold back this PR.

Reply:
Yeah, I removed them because the linter gave these errors:
```
  17:63  error  Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead  @typescript-eslint/ban-types
```
And then I thought `React.FunctionComponent<Record<string, never>>` looked way too ugly for something so routine/simple, so I looked up some examples of how to write a TS function component and saw that those return types are unnecessary. 😄

Reply:
This one is my favorite:
```
const App = ({ message }: AppProps): JSX.Element => <div>{message}</div>;
```
because at least it says it returns JSX.Element

Reply:
Ooh I like that. I think I should make a separate issue/PR for that though, because it'd be best to go through all the function components (the lint fixes in this PR only touch those without any props) and make them all use it.